scio-examples/src/main/java/org/apache/beam/examples/common/ExampleCloudSqlOptions.java

```diff
@@ -0,0 +1,46 @@
+package org.apache.beam.examples.common;
+
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.Validation;
+
+public interface ExampleCloudSqlOptions extends PipelineOptions {
+
+  /**
+   * Get the username as command line option
+   */
+  @Description("Cloud Sql database username")
```

Comment:
Stylize `Sql` as `SQL` in docs everywhere?
This class is an example so you don't have to go crazy with JavaDocs. They're kinda redundant and just eyesore.